backend/core/src/migration/1635889621244-seed-detroit-translation-entries.ts

```diff
@@ -0,0 +1,68 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
Wait so there's no actual db change here? Were we just missing a migration?

Reply:
The "migration" (agree that it's not _really_ a migration, it's literally just adding a row) that was doing this was only inserting a row for Alameda. When we were trying to send emails with the "Detroit" jurisdiction, TranslationsService couldn't find anything.

I considered adding an entry to `detroit-seed.ts`, but figured it would be faster to follow the existing pattern.